src/marketplace/eqp/v1/auth.md

```diff
@@ -47,15 +47,17 @@ You must specify the grant type in the request body:
 
 ```json
 {
-   "grant_type": "session"
+   "grant_type": "session",
+   "expires_in": 7200
 }
 ```
 
 Field details:
 
-|Field|Type|Description|
-|-----|----|-----------|
-|grant_type|string|The API only supports the `session` grant type; other values will give an error.|
+|Field|Type|Required|Description|
+|-----|----|----|-----------|
+|grant_type|string|yes| The API only supports the `session` grant type; other values will give an error.|
+|expires_in|int|no| Requested time in seconds that the session token will be valid. If the requested time exceeds the system's maximum allowed, the system's maximum limit will be used instead. |
```

Comment:
```suggestion
|expires_in|int|no| Specifies the number of seconds that the session token will be valid. If the requested time exceeds the system's maximum allowed, the system's maximum limit will be used instead. |
```

Reply:
I rewrote the passage because I had trouble parsing "requested time in seconds". Until I looked at the example,  I wasn't sure whether this meant a UNIX time stamp for some future hour or the raw number of seconds. Feel free to correct this if I missed some subtlety.